client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -0,0 +1,319 @@
+import { getManagePurchaseUrlFor } from 'calypso/my-sites/purchases/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
+class EmailPlan extends React.Component {
+	static propTypes = {
+		domain: PropTypes.object.isRequired,
+		selectedSite: PropTypes.object.isRequired,
+
+		// Connected props
+		currentRoute: PropTypes.string,
+		hasEmailPlanSubscription: PropTypes.bool,
+		isLoadingPurchase: PropTypes.bool,
+		purchase: PropTypes.object,
+	};
+
+	state = {
+		isLoadingEmailAccounts: false,
+		errorLoadingEmailAccounts: false,
+		loadedEmailAccounts: false,
+		emailAccounts: [],
+	};
+
+	componentDidMount() {
+		this.loadEmailAccounts();
+	}
+
+	componentDidUpdate() {
+		this.loadEmailAccounts();
+	}
+
+	loadEmailAccounts() {
+		const { domain, selectedSite } = this.props;
+
+		if ( this.state.isLoadingEmailAccounts || this.state.loadedEmailAccounts ) {
+			return;
+		}
+
+		this.setState( {
+			isLoadingEmailAccounts: true,
+		} );
+
+		wp.undocumented()
+			.getEmailAccountsForSiteAndDomain( selectedSite.ID, domain.name )
+			.then(
+				( data ) => {
+					this.setState( {
+						isLoadingEmailAccounts: false,
+						errorLoadingEmailAccounts: false,
+						loadedEmailAccounts: true,
+						emailAccounts: data?.accounts || [],
+					} );
+				},
+				() => {
+					this.setState( {
+						isLoadingEmailAccounts: false,
+						errorLoadingEmailAccounts: true,
+						loadedEmailAccounts: true,
+						emailAccounts: [],
+					} );
+				}
+			);
+	}
+
+	getEmails() {
+		if ( this.state.emailAccounts[ 0 ] ) {
+			return this.state.emailAccounts[ 0 ].emails;
+		}
+		return [];
+	}
+
+	handleBack = () => {
+		const { selectedSite } = this.props;
+		page( emailManagement( selectedSite.slug ) );
+	};
+
+	getAddMailboxProps() {
+		const { currentRoute, domain, selectedSite } = this.props;
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return {
+				path: emailManagementAddGSuiteUsers(
+					selectedSite.slug,
+					domain.name,
+					getProductType( getGoogleProductSlug( domain ) ),
+					currentRoute
+				),
+			};
+		}
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			const subscriptionId = getTitanSubscriptionId( domain );
+			return {
+				external: ! subscriptionId,
+				path: emailManagementNewTitanAccount( selectedSite.slug, domain.name, currentRoute ),
+			};
+		}
+
+		return {
+			path: emailManagementForwarding( selectedSite.slug, domain.name, currentRoute ),
+		};
+	}
+
+	getHeaderText() {
+		const { domain, translate } = this.props;
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			const googleMailService = getGoogleMailServiceFamily( getGoogleProductSlug( domain ) );
+			return translate( '%(googleMailService)s settings', {
+				args: {
+					googleMailService,
+				},
+				comment: '%(googleMailService)s can be either "GSuite" or "Google Workspace"',
+			} );
+		}
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			return translate( 'Email settings' );
+		}
+
+		return translate( 'Email forwarding settings' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Email Forwarding' )` *ES Score: 6*